spark-on-watson-studio/nested-json-to-table.md

```diff
@@ -0,0 +1,64 @@
+# Extract and save nested JSON elements into separate Db2 tables
+This example extracts nested elements from a JSON array and saves each array item into new Db2 tables.  
+
+**Note:** this is an introductory example and does not include any Db2 primary key-foreign key table relationships.
+
+The example below will use this JSON document:
+```
```

Comment:
can you add the annotation here for json syntax highlighting i.e.  ` ```json`

Reply:
Added in 00d7123.